src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_pending_resumption_handler.cc

```diff
@@ -77,6 +90,7 @@ void RCPendingResumptionHandler::HandleResumptionSubscriptionRequest(
     const auto resumption_request =
         MakeResumptionRequest(cid, fid, *subscription_request);
     freezed_resumptions_[subscription].push(resumption_request);
+    application_manager_.set_application_id(cid, app.app_id());
```

Comment:
why do we need this?

Reply:
We need to get the app after receiving the response from the HMI and cid is only the way to detect the app, but firstly we need to remember the appropriate cid and app_id, but if we will use rc_cache inside the rc_pending_resumption - all this staff will be redundant